test/acceptance/behave/components/ngsiv2/entities/list_entities/list_entities_queries_parameters_part_3.feature

```diff
@@ -31,8 +31,8 @@
 
 Feature: list all entities with get request and queries parameters using NGSI v2. "GET" - /v2/entities/
   Queries parameters
-     tested : limit, offset, id, idPattern, type, q and option=count,keyValues
-     pending: georel, geometry, coords and option=values,distinct
+  tested : limit, offset, id, idPattern, type, q and option=count,keyValues
+  pending: georel, geometry, coords and option=values,distinct
```

Comment:
At the end, we are going to use "unique" instead of "distinct" in NGSIv2.

Please, do an overall "replace all" for this keyword in everyplace.